data/BGC0001599.json

```diff
@@ -25,7 +34,20 @@
         ],
         "compounds": [
             {
-                "chem_struct": "CCCCCCCC(=O)NCC(C(C)C)N(N=O)O",
+                "chem_struct": "CCCCCCCC(=O)NCC(C(C)C)/[N+](=N/O)/[O-]",
+                "chem_synonyms": [
```

Comment:
Maybe we should just use  "octanamide" and drop the rest.

Reply:
Done